app/views/spotlight/exhibits/_exhibits.html.erb

```diff
@@ -0,0 +1,4 @@
+# Overrides partial to allow display of four exhibit cards in a row.
```

Comment:
Should get rid of this, it's displaying.

Reply:
Fixed.